models/event_argument_extraction/Linker.py

```diff
@@ -0,0 +1,45 @@
+from EntLink.api import init, Related, filter
+import spacy
+
+class Linker(object):
+    def __init__(self):
+        self.path_to_ckpt = ''
+        self.tokenizer = None
+        self.model = None
+        self.model_para = dict()
+
+    def instantiate(self):
+        pass
+         
+    def forward(self):
+        pass
+
+
+class RelationLinker(object):
+
+    def instantiate(self):
+        init()
+         
+    def forward(self, text:str) -> list:
+        return filter(Related(text))
```

Comment:
ah, so the linker class is here. I would then put the methods above (in the api.py) inside this class, since the methods are used only in this RelationLinker. (or if it is also used in other Linkers, they should be in the base class)

Reply:
done! :)